### PR TITLE
Main page styling 

### PR DESCRIPTION
**Pull request by J-0987**

### CODE REVIEW

File: assets/js/script.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+
+const bodySelector =  document.querySelector('body');
+const gameSection = document.querySelector('#game-section')
+
```
Comment (BryanPeens) [inline assets/js/script.js:4]: I noticed the start button seems missing
```suggestion
const startBtn = document.querySelector('#start-button');
```

File: assets/js/script.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+//Update time
+
+function displayTime () {
+  const today = dayjs();
+  $('#day').text(today.format('dddd, MM/D/YY, h:mm:ss a'));
+} 
+setInterval(displayTime, 1000);
+
+// // method 1
```
Comment (BryanPeens) [inline assets/js/script.js:13]: Is this needed or can the comments be removed?

**Review by BryanPeens — COMMENTED** [review]
Looks good to me!

File: assets/js/script.js (hunk 2 above)
Comment (J-0987) [reply assets/js/script.js:13]: Can be removed!

File: assets/js/script.js (hunk 1 above)
Comment (J-0987) [reply assets/js/script.js:4]: oh wow, i deleted it by accident, thank you Bryan!

**Review by BryanPeens — APPROVED** [review]

**Pull request merged**